crawl-ref/Makefile

```diff
@@ -0,0 +1,3 @@
+make_webtiles:
```

Comment:
we won't merge this makefile, can you remove it from the PR? I understand it may be convenient, but it's really ad hoc, and sudo *very very much* should not be present in a Makefile